core/templates/dev/head/pages/admin-page/activities-tab/admin-dev-mode-activities-tab.directive.ts

```diff
@@ -66,57 +67,68 @@ oppia.directive('adminDevModeActivitiesTab', [
           });
         };
 
-        ctrl.DEMO_EXPLORATIONS = GLOBALS.DEMO_EXPLORATIONS;
-        ctrl.DEMO_COLLECTIONS = GLOBALS.DEMO_COLLECTIONS;
         ctrl.numDummyExpsToPublish = 0;
         ctrl.numDummyExpsToGenerate = 0;
-
+        ctrl.DEMO_COLLECTIONS = {};
+        ctrl.DEMO_EXPLORATIONS = {};
+        var canReloadAllExplorations = false;
+        var demoExplorationIds = [];
         ctrl.reloadAllExplorations = function() {
-          if (AdminTaskManagerService.isTaskRunning()) {
-            return;
-          }
-          if (!$window.confirm('This action is irreversible. Are you sure?')) {
-            return;
-          }
-
-          ctrl.setStatusMessage('Processing...');
-          AdminTaskManagerService.startTask();
+          if (canReloadAllExplorations) {
```

Comment:
Instead of moving all the code inside into if block, maybe you can do :
```
if (!canReloadAllExplorations) {
    return;
}

```

Also, can you rename the variable to `reloadingAllExplorationPossible` or something around this? (In general, let's not start the name of a boolean variable with the verb. We use to do this with function names.)

Reply:
Done